debugger/gdb-extension/gdb_sgx_plugin.py

```diff
@@ -292,7 +296,7 @@ def stop(self):
         module_addr = int(gdb.parse_and_eval("$rdi"))
         debug_module = oe_debug_module_t(module_addr)
         load_enclave_symbol(debug_module.path, debug_module.base_address)
-        print ("oegdb: Loaded enclave module %s" % debug_module.path)
+        #print ("oegdb: Loaded enclave module %s" % debug_module.path)
```

Comment:
Is this intended?

Reply:
Somebody mentioned that these messages were too noisy.
Do you find these messages useful?

Reply:
Should we just get rid of it then?

Reply:
Personally I find the messages useful as anchor points in a debugging sessions. I was thinking of creating a verbose option for the debugger, but didn't get to it yet.

Reply:
I'm fine with it. We can do that later.